libbeat/_meta/config/output-elasticsearch.reference.yml.tmpl

```diff
@@ -81,6 +81,11 @@ output.elasticsearch:
   # Elasticsearch after a network error. The default is 60s.
   #backoff.max: 60s
 
+  # The maximum amount of time an idle connection will remain idle
+  # before closing itself.  Zero means no limit. The format is a Go
```

Comment:
The wording here states zero is no limit. But in the conversation and snippet [here](https://github.com/elastic/beats/pull/36843#discussion_r1367176297), isn't setting `idle_connection_timeout` to `0s` going to set the client to still use a 60s timeout?

Reply:
Thanks, should be fixed now.